### PR TITLE
#322 : Right Sidebar dark theme issue resolved

### PR DESCRIPTION
**Pull request by 1mazin**
## Description 📝
The right sidebar in the dashboard is now supporting dark theme.

## Screenshots

Before
image

After
changed

## Additional Notes 📒
@PiyushKalyanpy I want to contribute under gssoc23

### CODE REVIEW

File: Components/RightSideBar/Calendar.tsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ const Calendar = (props: Props) => {
   return (
     <div className="mb-6">
       <h1 className="font-bold mb-2">Calendar</h1>
-      <div className="w-full bg-[#F0F4F8] rounded-lg px-4 py-2">
+      <div className="w-full dark:bg-zinc-800 bg-[#F0F4F8] rounded-lg px-4 py-2">
         <div className="flex flex-row justify-around gap-2">
           <IndividualDate date={12} day="M" isTodayDate={false} />
           <IndividualDate date={13} day="T" isTodayDate={false} />
```
Comment (PiyushKalyanpy) [inline Components/RightSideBar/Calendar.tsx:13]: Update the color classes for these buttons also, when hovering a button.

**Review by PiyushKalyanpy — CHANGES_REQUESTED** [review]

**Comment by 1mazin** [comment]
Could you please check now.

File: database/firebase.tsx
Comment (PiyushKalyanpy) [inline database/firebase.tsx:1]: remove this file from PR

File: package-lock.json
Comment (PiyushKalyanpy) [inline package-lock.json:1]: remove this file

**Review by PiyushKalyanpy — CHANGES_REQUESTED** [review]

**Comment by PiyushKalyanpy** [comment]
Please create a new pr, I think by mistake you have pushed those files which are not required and removed the required files.

**Comment by 1mazin** [comment]
I have created a new pull request for the same. Could you please review that.

**Pull request closed without merging**